pkg/config/load.go

```diff
@@ -43,6 +51,8 @@ type Info struct {
 	Variant string
 	// Filename is the full path to the file on disk
 	Filename string
+
+	ProwgenConfig *ProwgenConfig
```

Comment:
There's no need to use a pointer here, a zero-value struct will have a `false` boolean and will work fine.